### PR TITLE
Monitor layout

### LINKED ISSUE #3: Create User page
## Desktop
user page

### LINKED ISSUE #5: Create Organization page
## Desktop
Frame 10

### LINKED ISSUE #22: Create login page
Create component with login layout and paste it into login page and dialog.

### LINKED ISSUE #23: Create registration page
Create component with registration form and paste it into registration page and dialog.

### LINKED ISSUE #25: Fix warnings in console
# Overview
__Fix all warnings and errors__, displayed in console when starting program!!! 
Also, __remove__ all unused __console logs__.
# Images
## Errors and other trash in console
image
## Extra logs
image

### LINKED ISSUE #26: Create Auth context
# Overview
Create ___Auth___ context for handling login and token storing. It should contain functions:
* ```getUser()```
* ```isLogged```
* ```login()```
* ```logout()```

### LINKED ISSUE #27: Connect Auth context with coreRequest
Create connection between __Rest requests__ and __Auth system__.

### PR DESCRIPTION
**Pull request by ZaykaNya**
Created all pages except Submit Page.

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_CORE_URL=https://pathfinder-core.herokuapp.com
```
Comment (DanilAndreev) [inline .env.development]: Change to GCloud core: 34.121.59.155:3002

File: package.json [hunk 2]
```diff
@@ -38,15 +38,22 @@
...
   },
   "dependencies": {
+    "@date-io/date-fns": "1.x",
     "@devexpress/dx-react-chart": "^2.7.2",
     "@devexpress/dx-react-chart-material-ui": "^2.7.2",
     "@devexpress/dx-react-core": "^2.7.2",
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
+    "@material-ui/pickers": "^3.2.10",
+    "@types/prop-types": "^15.7.3",
```
Comment (DanilAndreev) [inline package.json]: Types to dev dependencies

File: src/components/DataTextField/DataTextField.tsx [hunk 3]
```diff
@@ -39,25 +38,26 @@ interface DataTextFieldProps extends Stylable, Containerable{
...
-       <Box className={clsx(classes.boxContainer, className)}>
-           <Typography className={clsx(classes.boxContainerTitle, className)}>
-               {label}
-           </Typography>
-           <Typography className={clsx(classes.boxContainerText, className)}>
-               {children}
-           </Typography>
-       </Box>
-   );
+    const ref1 = React.useRef<any>();
```
Comment (DanilAndreev) [inline src/components/DataTextField/DataTextField.tsx]: name variables so, that everyone can understand what is this.

File: package.json [hunk 4]
```diff
@@ -38,15 +38,22 @@
...
+    "date-fns": "^2.16.1",
+    "notistack": "^1.0.1",
+    "prop-types": "^15.7.2",
     "moment": "^2.29.1",
```
Comment (DanilAndreev) [inline package.json]: Leave one of: moment or datefns

File: src/components/FileField/FileField.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+ */
+const FileField = React.forwardRef((props: FileFieldProps, ref: React.Ref<any>) => {
+    const {
+        classes,
+        className
+    } = props;
+
+    return(
+        <Box className={classes.root}>
+            <input
```
Comment (DanilAndreev) [inline src/components/FileField/FileField.tsx]: Change to MaterialUI InputBase. Do not use raw html

File: src/components/FileField/styles.ts [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+ * Project: pathfinder-monitor
+ * File last modified: 07.10.2020, 18:22
+ * All rights reserved.
+ */
+
+import {createStyles, Theme} from "@material-ui/core";
+
+const styles = (theme: Theme) => createStyles({
+    root: {
+        '& > *': {
```
Comment (DanilAndreev) [inline src/components/FileField/styles.ts]: double quotes

File: src/components/DataTextField/DataTextField.tsx (hunk 3 above)
Comment (DanilAndreev) [reply src/components/DataTextField/DataTextField.tsx]: Remove unused refs!

File: src/views/AuthorizationPageView/AuthorizationPageView.tsx [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+import Container from '@material-ui/core/Container';
+import Stylable from "../../interfaces/Stylable";
+import {withStyles} from "@material-ui/core";
+import styles from "./style"
+import useCoreRequest from "../../hooks/useCoreRequest";
+import useAuth from "../../hooks/useAuth";
+import {useChangeRoute} from "routing-manager";
+import { useSnackbar } from 'notistack';
+
+interface AuthorizationPageViewPropsStyled extends Stylable {
```
Comment (DanilAndreev) [inline src/views/AuthorizationPageView/AuthorizationPageView.tsx]: Remove Styled postfix from name

File: src/views/AuthorizationPageView/AuthorizationPageView.tsx [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+        <Container component="main" maxWidth="xs">
+            <CssBaseline/>
+            <Box className={classes.paper}>
+                <Avatar className={classes.avatar}>
+                    <LockOutlinedIcon/>
+                </Avatar>
+                <Typography component="h1" variant="h5">
+                    Sign in
+                </Typography>
+                <form className={classes.form} noValidate onSubmit={handleLogin}>
```
Comment (DanilAndreev) [inline src/views/AuthorizationPageView/AuthorizationPageView.tsx]: Replace html form to MaterialUI component. Or remove form fully.

File: src/views/OrganizationPageView/LocalComponents/PluginComponent/PluginComponent.tsx [hunk 9]
```diff
@@ -20,7 +30,7 @@ import Stylable from "../../../../interfaces/Stylable";
  * @interface
  * @author Nikita Nesterov
  */
-interface PluginComponentPropsStyled extends Stylable{
+interface PluginComponentPropsStyled extends Stylable {
```
Comment (DanilAndreev) [inline src/views/OrganizationPageView/LocalComponents/PluginComponent/PluginComponent.tsx]: Remove Styled from name

File: src/views/OrganizationPageView/LocalComponents/PluginComponent/PluginComponent.tsx [hunk 10]
```diff
@@ -66,6 +80,38 @@ const PluginComponent = React.forwardRef((props: PluginComponentPropsStyled, ref
                     </Grid>
                 </Grid>
             </Grid>
+        )
+    }
+    else{
+        item=(
```
Comment (DanilAndreev) [inline src/views/OrganizationPageView/LocalComponents/PluginComponent/PluginComponent.tsx]: Where are spaces? Tap ctrl + alt + L or hook thi commant to save event in webstorm!

File: src/views/OrganizationPageView/LocalComponents/PluginComponent/PluginComponent.tsx [hunk 11]
```diff
@@ -39,11 +49,14 @@ const PluginComponent = React.forwardRef((props: PluginComponentPropsStyled, ref
         description,
     } = props;
 
-    return (
-        <ListItem>
+    const theme = useTheme();
+    const matches = useMediaQuery(theme.breakpoints.up("md"));
+    let item;
+    if(matches){
+        item=(
```
Comment (DanilAndreev) [inline src/views/OrganizationPageView/LocalComponents/PluginComponent/PluginComponent.tsx]: The same, spaces. Code style

File: src/views/OrganizationPageView/OrganizationPageView.tsx [hunk 12]
```diff
@@ -36,8 +37,15 @@ import Stylable from "../../interfaces/Stylable";
  * @interface
  * @author Nikita Nesterov
  */
-interface OrganizationPageViewPropsStyled extends Stylable{
+interface OrganizationPageViewPropsStyled extends Stylable {
```
Comment (DanilAndreev) [inline src/views/OrganizationPageView/OrganizationPageView.tsx]: minus  Styled

File: src/views/SignUpPage/SignUpPage.tsx [hunk 13]
```diff
@@ -0,0 +1,171 @@
...
+        <Container component="main" maxWidth="xs">
+            <CssBaseline/>
+            <Box className={classes.paper}>
+                <Avatar className={classes.avatar}>
+                    <LockOutlinedIcon/>
+                </Avatar>
+                <Typography component="h1" variant="h5">
+                    Sign up
+                </Typography>
+                <form className={classes.form} noValidate onSubmit={handleRegister}>
```
Comment (DanilAndreev) [inline src/views/SignUpPage/SignUpPage.tsx]: Form! Remove or replace with MaterialUI

File: src/views/SignUpPage/SignUpPage.tsx [hunk 14]
```diff
@@ -0,0 +1,171 @@
...
+                        <Grid item>
+                            <Link href="#" variant="body2">
+                                Already have an account? Sign in
+                            </Link>
+                        </Grid>
+                    </Grid>
+                </form>
+            </Box>
+            <Box mt={5}>
+                {/*<Copyright />*/}
```
Comment (DanilAndreev) [inline src/views/SignUpPage/SignUpPage.tsx]: remove that

File: src/views/UserPageView/LocalComponents/OrganizationsFieldsRow/OrganizationsFieldsRow.tsx [hunk 15]
```diff
@@ -20,7 +20,7 @@ import Stylable from "../../../../interfaces/Stylable";
  * @interface
  * @author Nikita Nesterov
  */
-interface OrganizationsFieldsRowPropsStyled extends Stylable{
+interface OrganizationsFieldsRowPropsStyled extends Stylable {
```
Comment (DanilAndreev) [inline src/views/UserPageView/LocalComponents/OrganizationsFieldsRow/OrganizationsFieldsRow.tsx]: Remove Styled postfix

File: src/views/UserPageView/LocalComponents/OrganizationsFieldsRow/OrganizationsFieldsRow.tsx [hunk 16]
```diff
@@ -31,28 +31,54 @@ interface OrganizationsFieldsRowPropsStyled extends Stylable{
...
-                <DataTextField label="Organization" children={organization}/>
-            </Grid>
-            <Grid item xs={2}>
-                <DataTextField label="Role" children={role}/>
+    const theme = useTheme();
+    const matches = useMediaQuery(theme.breakpoints.up("md"));
+    let Row;
+
+    if(matches){
+        Row = (
```
Comment (DanilAndreev) [inline src/views/UserPageView/LocalComponents/OrganizationsFieldsRow/OrganizationsFieldsRow.tsx]: why row starts with capital letter.

File: src/views/UserPageView/LocalComponents/TokensViewer/TokensViewer.tsx [hunk 17]
```diff
@@ -30,7 +20,7 @@ import Stylable from "../../../../interfaces/Stylable";
  * @interface
  * @author Nikita Nesterov
  */
-interface TokensViewerPropsStyled extends Stylable{
+interface TokensViewerPropsStyled extends Stylable {
```
Comment (DanilAndreev) [inline src/views/UserPageView/LocalComponents/TokensViewer/TokensViewer.tsx]: Remove Styled postfix

File: src/views/UserPageView/LocalComponents/TokensViewer/TokensViewer.tsx [hunk 18]
```diff
@@ -48,35 +38,54 @@ const TokensViewer = React.forwardRef((props: TokensViewerPropsStyled) => {
 
     const [isOpen, setIsOpen] = React.useState(false);
 
-    const handleClick=()=>{
+    const handleClick = () => {
```
Comment (DanilAndreev) [inline src/views/UserPageView/LocalComponents/TokensViewer/TokensViewer.tsx]: make this function single line

**Review by DanilAndreev — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (ZaykaNya) [reply package.json]: Fixed

File: package.json (hunk 4 above)
Comment (ZaykaNya) [reply package.json]: leaved datefns

**Review by DanilAndreev — APPROVED** [review]

**Pull request merged**